### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by rpask00** [hluk/CopyQ#1941] [Q&A]
Hi, i am facing an issue with an program, everything used to worked perfectly fine, and suddenly all copied elements are just big white squares without text.
![image](https://user-images.githubusercontent.com/33097890/159986454-c7a25b68-4674-4d49-b9f8-81d2a69fae42.png)

there are also long scrollbars..

reinstalling doesn't help

![copyq](https://user-images.githubusercontent.com/33097890/159986797-b9c3b5f4-20be-4f25-9ea4-e59c9da57fd3.gif)

does anyone know how to fix it?

**Comment by rpask00** [comment]
It is on ubunut if that matters.

**Comment by hluk** [comment]
See #1728.
